src/wind_energy/FreeStream.H

```diff
@@ -0,0 +1,44 @@
+#ifndef FreeStream_H
```

Comment:
I think we should move physics stuff out of `wind_energy` into its own `physics` directory. Thoughts?